src/services/pact-factory-service.test.ts

```diff
@@ -30,7 +30,7 @@ describe('PactFactory', () => {
 
   describe('Given a PactFactory service', () => {
     describe("When calling 'createContractBetween' method", () => {
-      let pact;
+      let pact: any;
```

Comment:
Would be better to use the appropriate type here if you can

Reply:
Updated to correct type! Thanks :)